src/Layout/tool_fsharp/tool_fsc.csproj

```diff
@@ -5,13 +5,31 @@
     <ServerGarbageCollection>true</ServerGarbageCollection>
   </PropertyGroup>
 
+  <!-- MIBC, PGO -->
+  <PropertyGroup>
+    <MibcTargetOS>linux</MibcTargetOS>
+    <!-- We deliberately want to use linux mibc pgo data on macOS -->
+    <MibcTargetOS Condition="$([MSBuild]::IsOSPlatform('OSX'))">linux</MibcTargetOS>
```

Comment:
When the sdk builds it builds cross target --- ie the tooling used to build the product doesnot necessarily match the build target.

When you do mibc you need to match the build target not the tooling platform.

Reply:
Hmmm alright, I am trying to leverage NETCoreSdkRuntimeIdentifier now - is this the right thing?

Reply:
Probably not :)